### ISSUE TITLE
sft微调样本，图片和视频同时放一个样本里，报错呢

### ISSUE THREAD

**Issue by zkailinzhang** [QwenLM/Qwen3-VL#1887]
模型Qwen3-VL-8B-Instruct，这样设置数据 就报错呢，数据格式
``` 
[
   {
      "image": "demo/images/10149.png",
      "video": "demo/videos/v_rBMQFpHspmo.mp4",
      "conversations": [
        {
          "from": "human",
          "value": "In which year the value was 51?\n<image><video>"
        },
        {
          "from": "gpt",
          "value": "2014"
        }
      ]
    },
]
__init__.py
multi_modal_v0 = {
    "annotation_path": "/appdata/Qwen3-VL-new/qwen-vl-finetune/demo/multi_modal_v0.json",
    "data_path": "/appdata/Qwen3-VL-new/qwen-vl-finetune/",
}

xxx.sh
    --max_pixels 50176 \
    --min_pixels 784 \
    --video_fps 2 \

```
错误信息：
[Try #0] Failed to fetch sample 2. Exception: Incorrect image source. Must be a valid URL starting with `http://` or `https://`, a valid path to an image file, or a base64 encoded string. Got /appdata/Qwen3-VL-new/qwen-vl-finetune/demo/images/100095.png. Failed with cannot identify image file <_io.BytesIO object at 0x7f73331b1300>
[2025-12-03 15:01:56,170] [WARNING] [lr_schedules.py:686:get_lr] Attempting to get learning rate from scheduler before it has started
[Try #1] Failed to fetch sample 2. Exception: Incorrect image source. Must be a valid URL starting with `http://` or `https://`, a valid path to an image file, or a base64 encoded string. Got /appdata/Qwen3-VL-new/qwen-vl-finetune/demo/images/100095.png. Failed with cannot identify image file <_io.BytesIO object at 0x7f73331b1300>
